### ISSUE TITLE
Durable Functions extension broken in latest Azure Functions v2 runtime

### ISSUE THREAD

**Issue by cgillum** [Azure/azure-functions-durable-extension#210]
### Issue

On 3/26/2018, an Azure Functions v2 runtime update was deployed to West US 2.  See [this announcement](https://github.com/Azure/app-service-announcements/issues/94) for more details.  Unfortunately this update contains several breaking changes for various extensions, including the Durable Functions extension.

### Symptoms

The breaking changes will prevent the Functions host from starting if the Durable Functions extension is loaded. The error message you can expect to see looks something like this:

```
The function runtime is unable to start. Microsoft.Azure.WebJobs.Extensions.DurableTask: Method not found: 'Microsoft.Azure.WebJobs.Host.TraceWriter Microsoft.Azure.WebJobs.Host.Config.ExtensionConfigContext.get_Trace()'.
Session Id: e4250f6d669f479b9535a2b676341502

Timestamp: 2018-03-26T23:18:31.462Z
```

This issue will require an update to the Durable Functions extension before it can be used in the latest Functions v2 runtime.

### Workaround(s)
Use Functions v1 instead of v2 (Durable Functions supports both runtime versions).

**Comment by GregLevenhagen** [comment]
Did this hist North Central US in the last day? Not getting the same error message, but it is an issue with DurableTask.

**Comment by cgillum** [comment]
Yes, I believe North Central US was updated yesterday. Just to confirm, what error message are you seeing?

**Comment by GregLevenhagen** [comment]
```
2018-03-28T14:22:50.888 [Information] Loaded custom extension: DurableTaskExtension from 'referenced by: Method='<assembly namespace>.HttpStart', Parameter='starter'.'
2018-03-28T14:22:50.890 [Trace] Extension loading complete.
2018-03-28T14:22:52.133 [Error] A ScriptHost error has occurred
2018-03-28T14:22:52.224 [Information] Stopping Host
```

Just started with our nightly process early in the AM today. No further details.

**Comment by cgillum** [comment]
This has been fixed in the [Beta 3.1 release (v1.2.1-beta3)](https://github.com/Azure/azure-functions-durable-extension/releases/tag/v1.2.1-beta3).
